RNGoogleMobileAdsExample/App.tsx

```diff
@@ -199,7 +233,7 @@ class InterstitialTest implements AutoExecutableTest {
 }
 
 class BannerTest implements AutoExecutableTest {
-  bannerRef: RefObject<BannerAd>;
+  bannerRef: RefObject<BannerAd | null>;
```

Comment:
this appears to be the possible impact of that null change prior as I forward-ported to modern types